### DISCUSSION TITLE
Multi factor authentication

### DISCUSSION THREAD

**Discussion by churnd** [homebridge-simplisafe3/homebridge-simplisafe3#213] [Q&A] [answered]
The app is now telling us that multi factor authentication will be required soon.  I'm assuming setting it up will break this plugin?

![image](https://user-images.githubusercontent.com/683720/117443424-1dd38c80-af06-11eb-8175-a714a3b1ac62.png)

**Comment by sterumbelow** [comment]
Shouldn't do. It's for authenticating a new user. 

I've got it set up and my plug-in still works fine.

**Comment by shamoon** [comment, marked as answer]
This has existed for a long time and the plugin can already handle it. Its only needed when you startup the plugin first login. In that case the logs will report that you need to MFA, then you go click the verification link and that's it!
